blockdag/utxoset_test.go

```diff
@@ -356,51 +464,76 @@ func TestUTXODiffRules(t *testing.T) {
 			},
 		},
 		{
-			name: "equal outpoints different blue scores: empty this, first in toAdd and second in toRemove in other",
+			name: "empty this, first in toAdd in other",
 			this: &UTXODiff{
 				toAdd:    utxoCollection{},
 				toRemove: utxoCollection{},
 			},
 			other: &UTXODiff{
 				toAdd:    utxoCollection{outpoint0: utxoEntry1},
-				toRemove: utxoCollection{outpoint0: utxoEntry2},
+				toRemove: utxoCollection{},
 			},
 			expectedDiffFromResult: &UTXODiff{
 				toAdd:    utxoCollection{outpoint0: utxoEntry1},
-				toRemove: utxoCollection{outpoint0: utxoEntry2},
+				toRemove: utxoCollection{},
 			},
 			expectedWithDiffResult: &UTXODiff{
 				toAdd:    utxoCollection{outpoint0: utxoEntry1},
-				toRemove: utxoCollection{outpoint0: utxoEntry2},
+				toRemove: utxoCollection{},
 			},
 		},
 		{
-			name: "equal outpoints different blue scores: first in toAdd and second in toRemove in both this and other",
+			name: "empty this, one toRemove in other",
```

Comment:
Please add it to excel

Reply:
It was in excel named "empty this, first in toRemove in other".
Renamed in code, because this is the naming convention.